samples/app/src/main/res/values/themes.xml

```diff
@@ -10,12 +10,21 @@
         <item name="colorSecondaryVariant">@color/appyx_dark</item>
         <item name="colorOnSecondary">@color/appyx_bright</item>
         <!-- Status bar color. -->
-<!--        <item name="android:statusBarColor">?attr/colorPrimaryVariant</item>-->
+        <!--        <item name="android:statusBarColor">?attr/colorPrimaryVariant</item>-->
         <!-- Customize your theme here. -->
     </style>
 
     <style name="Theme.Appyx.NoActionBar">
         <item name="windowActionBar">false</item>
         <item name="windowNoTitle">true</item>
     </style>
+
+    <style name="Theme.Appyx.Starting" parent="Theme.SplashScreen">
+        <!-- Set the splash screen background -->
+        <item name="windowSplashScreenBackground">@color/appyx_dark</item>
```

Comment:
How does this work with dark/light theme like we do in the node at the moment?

Reply:
Good catch! I forgot to modify the theme for dark mode, is done now.